src/lib/context-menu.ts

```diff
@@ -255,6 +256,36 @@ export class ContextMenu {
         })
       )
     }
+    if (type === schema.nodes.table_element) {
+      let addGeneralNote = true
+      const tableElementFooter = findChildrenByType(
+        this.node,
+        schema.nodes.table_element_footer
+      )
+      if (tableElementFooter.length) {
+        const paragraphs = findChildrenByType(
```

Comment:
Alternatively, I think you can simply check the type of tableElementFooter[0].node.firstChild to see if it's a paragraph.

Reply:
done